### PR TITLE
ProgressBar

### PR DESCRIPTION
**Pull request by cdbkr**
a simple ProgressBar component

### CODE REVIEW

**Comment by morenoh149** [comment]
this looks useful

**Comment by qingfeng** [comment]
@cdbkr `react/addons` This package is deprecated？

File: lib/ui/ProgressBar.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	getDefaultProps: function() {
+		return {
+			progress: 0,
+			max: 100
+		};
+	},
+	render: function() {
+		return (
+			<div className="item-inner">
+				<progress
```
Comment (dcousens) [inline lib/ui/ProgressBar.js]: where is this `progress` magic?

**Comment by dcousens** [comment]
@qingfeng is it?  I know https://www.npmjs.com/package/react-addons is deprecated, but the suggestion is simple to use `react/addons` instead.

> Rather than use this package, simply use require('react/addons') in modules that require addons.

**Comment by jossmac** [comment]
Hi @cdbkr,

Thanks for the PR, looks good. Just a couple things:
- Probably shouldn't be inside an `item-inner` div
- Do you have some default styles for it you could add?

**Comment by qingfeng** [comment]
sorry, I was wrong @dcousens

**Comment by dcousens** [comment]
bump @jossmac

**Comment by jossmac** [comment]
Closing this for now, it's not ready

**Pull request closed without merging**